src/app/pages/MarginTradePage/components/MarginPagination/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { translations } from 'locales/i18n';
+import { Button, ButtonStyle } from 'app/components/Button';
+
+interface IMarginPaginationProps {
+  page: number;
+  loading: boolean;
+  isDisabled: boolean;
+  onChange: (value: number) => void;
+}
+
+export const MarginPagination: React.FC<IMarginPaginationProps> = ({
```

Comment:
The graph does not return the total count of a query, and there's currently no way for it to do that. So there isn't an equivalent for data.loans.length

Reply:
I know that graph doesn't return it but once we have all of the data, we know how many items we received. It's not ideal because we need to fetch everything at once but it is what it is.